firmware/hw_layer/backup_ram.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file	backup_ram.cpp
```

Comment:
would be amazing to extract  backup_ram refactoring into a separate PR. looks like refactoring is big enough to be on it's own